FeedStoreChallenge/Infrastructure/CoreData/ManagedCache.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  ManagedCache.swift
+//  FeedStoreChallenge
+//
+//  Created by Alexander on 17/5/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import CoreData
+
+@objc(ManagedCache)
+internal class ManagedCache: NSManagedObject {
```

Comment:
In Swift, `internal` is the default access level so it doesn't need to be set explicitly.

`internal` can be removed:

```suggestion 
class ManagedCache: NSManagedObject {
```

---

Please, make sure to remove it everywhere to make the code consistent.